Create 2025-Q3-GCP-WG.md (#514)
ossf/tac: TI-reports/2025/2025-Q3-GCP-WG.md

We also operate the "EU CRA Monthly Tech Talk" (née the "CRA Tech Bi-weekly"), the agenda of which is managed by the Awareness SIG.

We have a regular schedule of calls for our Awareness and Standards SIGs and have started to take minutes in our main minutes doc.  We have put the Tooling SIG on the back burner due to lack of energy at this time, as well as the fact that many of the goals of this SIG are actually being progressed in the ORBIT working group. We are operating in a similar mode to the Best Practices Working group, with our SIGs reporting into the main working group call.  We have well attended meetings. Our general working group call, besides being a place where SIGs report, also serves as a venue to work on general deliverables and to drive awareness with group members of related activities. 

> Last week, we held a special f2f hybrid meeting of the working group at Open Source Summit Europe. The focus on this meeting was to hear from two different organizations about their CRA readiness work and plans. We heard from Nokia and Balena, two very different companies. The results are minuted i[n our regular minutes doc](https://docs.google.com/document/d/1iAplSQheMgemdMnEw74uPj3oi_6rLLbFFXhg4svqIDo/edit?tab=t.0). We also discussed the recent call for comments from CISA on [their 2025 Minimum SBOM elements](https://www.cisa.gov/resources-tools/resources/2025-minimum-elements-software-bill-materials-sbom).

Reviewer: ```suggestion
Last week, we held a special f2f hybrid meeting of the working group at Open Source Summit Europe. The focus on this meeting was to hear from two different organizations about their CRA readiness work and plans. We heard from Nokia and Balena, two very different companies. The results are minuted [in our regular minutes doc](https://docs.google.com/document/d/1iAplSQheMgemdMnEw74uPj3oi_6rLLbFFXhg4svqIDo/edit?tab=t.0). We also discussed the recent call for comments from CISA on [their 2025 Minimum SBOM elements](https://www.cisa.gov/resources-tools/resources/2025-minimum-elements-software-bill-materials-sbom).
```

> We have helped to launch the successful [Free LF Training on CRA](https://openssf.org/press-release/2025/04/29/openssf-launches-free-course-to-prepare-developers-for-the-eu-cyber-resilience-act/) by providing feedback and input. ~3000 enrollments so far.  

Reviewer: These are great enrollment numbers!

## Awareness SIG

The awareness SIG is led by [Megan Knight](https://github.com/businesscasualkesha) of Arm. The scope is activities that drive awareness of the work of this group and of the regulatory landscape in general. The SIG has been marshalling blog posts, upcoming conference schedule, as well as the CRA introductory course. The Awareness SIG minutes are kept in the [main working group minutes document](https://docs.google.com/document/d/1iAplSQheMgemdMnEw74uPj3oi_6rLLbFFXhg4svqIDo/edit). The group is working on a CRA glossary.

Awareness sig has setup a project board for monthly content calendar organization: https://github.com/orgs/ossf/projects/33

Activities and Publications:
* Tech Talk on OSPS Baseline and how it support CRA
* Tech Talk on SBOM Toools and the CRA
* Tech Talk on Risk Assessment
* Tech Talk on "What makes you commercial" in the CRA
* Tech Talk o the Brief Guide, as well as [Blog Post on Brief Guide](https://openssf.org/blog/2025/07/15/new-cyber-resilience-act-cra-brief-guide-for-oss-developers/)

## Standards SIG

The Standards work stream is progressing. Since joining the team, Madalin has taken over leadership of this SIG.

> The SIGs mission was defined as: 

Author: Fixed. My bad.
